src/vl.js

```diff
@@ -4,7 +4,10 @@ var globals = require('./globals'),
     util = require('./util'),
     consts = require('./consts');
 
-var vl = util.merge(consts, util);
+var vl = {};  // util.merge(consts, util);
```

Comment:
I think splitting might be better, but then you should submit a concurrent pr in the other 3 projects for these then.